samples/meetings-sidepanel/csharp/Side Panel/Bots/SidePanelBot.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Teams;
+using Microsoft.Bot.Connector;
+using Microsoft.Bot.Connector.Teams;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Schema.Teams;
+using SidePanel.Controllers;
+
+namespace Microsoft.BotBuilderSamples.Bots
+{
+    public class SidePanelBot : TeamsActivityHandler
+    {
+        protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
+        {
+            string replyText;
+            var meetingId = turnContext.Activity.TeamsGetMeetingInfo().Id;
+            var userId = turnContext.Activity.From.Id;
+            var tenantID = turnContext.Activity.Conversation.TenantId;
+            string userName = turnContext.Activity.From.Name;
+
+            TeamsMeetingParticipant participant = await GetMeetingParticipantAsync(turnContext, meetingId, userId, tenantID);
+            TeamsChannelAccount member = participant.User;
+            MeetingParticipantInfo meetingInfo = participant.Meeting;
+            ConversationAccount conversation = participant.Conversation;
+            var meetingRole = meetingInfo.Role;
+
+            if(meetingRole == "Organizer")
+            {
+                HomeController.meetingOrganizerId = turnContext.Activity.From.AadObjectId;
```

Comment:
This part if bit confusing. Organizer need to send a message to Bot for this sample to work. Can we instead use Connector client directly to fetch participant details? Similar to this: https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/v-trkum/sidepanel/samples/meetings-sidepanel/csharp/Side%20Panel/Controllers/HomeController.cs#L85

Reply:
We tried to get the participants details but roles is not coming need to check this once

Reply:
Is this resolved?